client/flutter/pubspec.lock

```diff
@@ -64,6 +64,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "0.16.1"
+  firebase_messaging:
```

Comment:
@cbgrey can you double check that there weren't any changes to Podfile or Podfile.lock after you added these dependencies and then ran flutter on ios?

related -- we should probably get something in for https://github.com/WorldHealthOrganization/app/issues/647 so we can make sure we can check for this automatically